src/pages/outcomes/cost-and-carbon-savings.jsx

```diff
@@ -0,0 +1,220 @@
+  ],
+  imageSrc: SavingsImage1,
+  imageAlt: 'cilium big tcp stats',
+  imageRight: true,
+};
+
+const sectionContent3 = {
+  heading: 'High pod density with lean IP allocation',
+  paragraphs: [
+    'Particularly when used at scale, other CNIs may require large blocks of IP addresses for every node, limiting pod density and resulting in wasted resources. Because Cilium manages IP allocation more efficiently, you can run more pods per node without running out of IPs. This improves resource utilisation and decreases infrastructure and electricity costs by reducing the number of nodes needed.',
+  ],
+  videoSrc: 'https://www.youtube.com/embed/yikVhGM2ye8',
+  imageRight: false,
+};
+
+const sectionContent4 = {
+  heading: 'Eliminate cloud load balancer charges',
+  paragraphs: [
+    "With Cilium's Cluster Mesh, native pod-to-pod communication between clusters is possible without the need for costly cloud load balancers. This simplifies your architecture and prevents excessive data transfer fees. By removing external dependencies, teams gain more control over traffic, reduce latency, and cut down significantly on cloud networking costs.",
+  ],
+  imageSrc: SavingsImage2,
+  imageWidth: 624,
+  imageHeight: 399,
+  imageAlt: 'multi-cluster illustration',
+  imageRight: true,
+};
+
+const testimonials = [
+  {
+    logo: SmartnewsLogo,
+    title: 'Cutting networking overhead at scale',
+    CTAs: [
+      {
+        CTAtext: 'Read The Case Study',
+        url: 'https://www.cncf.io/case-studies/smartnews',
+      },
+    ],
+    description:
+      'SmartNews adopted Cilium to reduce the compute and cost impact of Kubernetes networking. With eBPF-based routing and automated policy enforcement, they reduced the number of nodes needed to run their workloads efficiently and improved resource efficiency, and direct savings on infrastructure and energy.',
+  },
+  {
+    title: 'Lowering network costs while debugging faster',
+    logo: KakaoLogo,
+    CTAs: [
+      {
+        CTAtext: 'Read The Case Study',
+        url: 'https://www.cncf.io/case-studies/kakao/',
+      },
+    ],
+    description:
+      'Kakao transitioned from kube-proxy to Cilium’s eBPF dataplane across 7,000 clusters and more than 120,000 nodes. This move cut network and performance costs, simplified operations by removing proxies, and sped up debugging workflows, resulting in fewer nodes and lower emissions.',
+  },
+  {
+    logo: SeznamLogo,
+    title: 'Lower costs with high-performance',
+
+    description:
+      'Seznam adopted Cilium to modernize its networking stack. The migration delivered improved network performance while also reducing both capital and operational costs, helping Seznam operate more efficiently at scale.',
+    CTAs: [
```

Comment:
you can also add the blog here

https://cilium.io/blog/2022/04/12/cilium-standalone-l4lb-xdp/